src/main/java/data/plans/PlanList.java

```diff
@@ -436,6 +459,29 @@ public boolean checkWorkoutInPlan(String workoutToCheck, Plan plan) throws Array
         return false;
     }
 
+    /**
+     * Gets the index number of plans which includes the updated workout.
+     * @param workoutToCheck        The workout to look for in the plan.
```

Comment:
Perhaps an extra line can be added between the method description and the @param?

Reply:
Thanks for spotting that!